packages/yarnpkg-core/sources/Project.ts

```diff
@@ -1579,8 +1599,8 @@ export class Project {
     for (const extensionsByIdent of this.configuration.packageExtensions.values())
       for (const [, extensionsByRange] of extensionsByIdent)
         for (const extension of extensionsByRange)
-          if (extension.active)
-            Configuration.telemetry?.reportPackageExtension(extension.description);
+          if (extension.userProvided && extension.status === `active`)
```

Comment:
I think it makes more sense to only report user-provided and non-redundant extensions inside our telemetry collection.

Reply:
I think it might actually make a lot of sense to report the redundant builtins, since it means we can remove them during the next major. Wdyt?

Reply:
I have a few concerns:
- We can't tell which builtins are ours and which are registered by custom (possibly private) plugins. We can only distinguish between user-provided and not user-provided.
- Redundant builtins are relative to a project and its installed dependency tree, since a builtin can match multiple packages and can be active for some of them and redundant for others (e.g. if a dependency issue gets fixed in a newer version of a package).
- We'd have to report redundant builtins separately to be able to distinguish them inside the dashboard from active user-provided extensions.